prime-router/src/main/kotlin/cli/ProcessFhirCommands.kt

```diff
@@ -162,7 +162,9 @@ class ProcessFhirCommands : CliktCommand(
             else -> {
                 val bundle = applySenderTransforms(fhirMessage)
                 FhirToHl7Converter(
-                    receiverSchema!!.name.split(".")[0], receiverSchema!!.parent,
+                    receiverSchema!!.name,
+                    // TODO: #10510
```

Comment:
#10510 is a general cleanup ticket after we complete this epic.  Trying to keep the scope of the code changes here as small as possible to make a reversion easy